cdap-ui/app/directives/log-viewer-preview/log-viewer-preview.js

```diff
@@ -302,6 +305,11 @@ function LogViewerPreviewController ($scope, $window, LogViewerStore, myPreviewL
     });
   };
 
+  vm.toggleSystemLogs = () => {
+    vm.includeSystemLogs = !vm.includeSystemLogs;
+    startTimeRequest();
```

Comment:
Currently on this branch preview doesn't have a default `fromOffset` yet, since that was introduced in the preview metrics PR which has yet to be merged.